tasks/volumes.yml

```diff
@@ -27,7 +27,7 @@
     {% endif %}
     -a {{ ansible_check_mode }}
   with_items: "{{ volumes }}"
-  when: item.type | default(libvirt_volume_default_type) == 'volume'
+  when: item.type | default(libvirt_volume_default_type) == 'volume' or ( item.type | default(libvirt_volume_default_type) == 'block' and item.pool is defined )
```

Comment:
I don't think this makes sense for a block device.

Reply:
Hi,
It's a workaround (only for block device in LVM) otherwise the tasks is skipped so the volume is not created.
as the variable need to be like this : 
```
  volumes:
    - name: 'ns2'
      type: 'block'
      capacity: '10GB'
      pool: 'lvm_pool'
      format: 'raw'
```

Reply:
I see, but I don't think it makes sense in the general case. Typically a block device would not be associated with a volume, AFAIU.

Reply:
In general case, volume.pool will not exist so it's will skip, so same behaviour than before